src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -30,16 +36,20 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   // Expanded
   --pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap: var(--pf-global--gutter);
 
+  // Chip container
+  --pf-c-data-toolbar--m-chip-container--MarginTop: calc(var(--pf-global--spacer--md) * -1);
```

Comment:
It looks like this modifier is on: `pf-c-data-toolbar__content` so the var name should be updated accordingly, same for the name below